src/mediaelement.js

```diff
@@ -178,6 +184,15 @@ export default class MediaElement extends WebAudio {
         this.isMuted = media.muted;
         this.setPlaybackRate(this.playbackRate);
         this.setVolume(this.volume);
+        if (this.params.mediaElementWebAudio) {
+            this.createMediaElementSource(media);
+        }
+    }
+    createMediaElementSource(mediaElement) {
+        this.sourceMediaElement = this.ac.createMediaElementSource(
+            mediaElement
+        );
+        this.sourceMediaElement.connect(this.analyser);
```

Comment:
An alternative option would be to keep WebAudioMedia as a separate backend and just inherit from the MediaElement backend. With just these two methods (`init` and `createMediaElementSource`).

Reply:
Yes, it can be also an alternative if we want handle audio with MediaElement backend only as HTML5 tag.  It is a business logic choice, I can restore to WebAudioMedia backend and inherit from MediaElement!

Reply:
I think it would be cleaner as a separate class. 👍

Reply:
But putting createMediaElementSource to this new class, i have to override _load method because it uses createMediaElementSource method.

Reply:
In ES6, you can use [`super`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super) to refer to the parent class methods.
So your method would looks smth like:
```
_load() {
  super._load();
  this.createMediaElementSource(media);
}
```

Reply:
ah thanks!